README.md

```diff
@@ -18,7 +18,7 @@ modules build.gradle file:
 ```groovy
 dependencies {
   compile 'com.segment.analytics.android:analytics:4.0.4'
-  compile 'com.leanplum.segment:LeanplumIntegration:{{LP_SEG_VERSION}}'
+  compile 'com.leanplum.segment:LeanplumIntegration:1.1.1-SNAPSHOT'
```

Comment:
yeah, in develop its supposed to be the next release version w/ the -SNAPSHOT postfix